schematic/schemas/data_model_relationships.py

```diff
@@ -188,6 +191,16 @@ def define_data_model_relationships(self) -> dict:
                     "standard": get_label_from_display_name,
                 },
             },
+            "columnType": {
+                "jsonld_key": "sms:columnType",
+                "csv_header": "ColumnType",
```

Comment:
Can you elaborate on what this impacts?

Reply:
Should this work for a csv version of data model with header `ColumnType` too? Or this feature is only intended for adding "columnType" to json-ld?

Reply:
It should eventually work for CSV as well. That's an issue @SageGJ  is working on.

Reply:
@thomasyu888 Two things: 
1. In the `DataModelJSONLDParser.gather_jsonld_attributes_relationships` method, it loops through all the entries from the JSON-LD, and for each of those loops through each relationship in the `DataModelRelationships`. For each entry/relationship it uses the relationship `jsonld_key` to see if the entry has that field, and if so, pull it out, parses it, and adds it to a dict of entries it found. This dict is eventually what is returned.
2. DataModelRelationships now has a new method `get_allowed_values` that gets the `allowed_values` for the relationship. `DataModelJSONLDParser.gather_jsonld_attributes_relationships` uses this, if it exists, to check the entry from the JSON-LD that it is in the `allowed_values` list.

Reply:
Thanks @andrewelamb !